### ISSUE TITLE
iojs@2.2.1 build error

### ISSUE THREAD

**Issue by ReklatsMasters** [bnoordhuis/node-heapdump#57]
Dalayload error

```
LINK : fatal error LNK1194: cannot delay-load "iojs.exe" due to import of data symbol ""__declspec(dllimpo rt) const v8::OutputStream::`vftable'" (__imp_??_7OutputStream@v8@@6B@)"; link without /DELAYLOAD:iojs.exe [path\to\node_modules\heapdump\build\addon.vcxproj]
```

**Comment by bnoordhuis** [comment]
Ah, I do believe that's @piscisaureus's handiwork, see https://github.com/nodejs/io.js/commit/efadffe.  Does this patch fix it?

```diff
diff --git a/binding.gyp b/binding.gyp
index 5ba08fe..5bfca66 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -16,6 +16,7 @@
   'targets': [
     {
       'target_name': 'addon',
+      'win_delay_load_hook': 'false',
       'sources': [
         'src/compat-inl.h',
         'src/compat.h',
```

**Comment by ReklatsMasters** [comment]
Yes, it works.

**Comment by bnoordhuis** [comment]
Thanks for testing.  Committed as 78056ec and published as v0.3.6.
